### ISSUE TITLE
THREE.Sprite: "Raycaster.camera" needs to be set in order to raycast against sprites.

### ISSUE THREAD

**Issue by lucychen0103** [pchen66/panolens.js#294]
##### Description

[Error] THREE.Sprite: "Raycaster.camera" needs to be set in order to raycast against sprites.
	raycast (three.min.js:2:377289)
	jl (three.min.js:2:550594)
	jl (three.min.js:2:550653)
	intersectObject (three.min.js:2:552052)
	outputPosition (panolens.min.js:181:238)

Does anyone know how to fix this? I am using three.min.js r105. 
Thanks! 

##### Panolens version

- [ ] Master
- [ ] Dev

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

**Comment by lucychen0103** [comment]
This is not solved but doesn't seem to have any effect on my situation. Thanks! Leaving it open in case someone else needs it solved.

**Comment by flyandi** [comment]
Yeah this is an old problem something that could be fixed with upgrading to an later threeejs library but that's not possible without changing some other stuff around at the moment.

**Comment by Nacbotics-dev** [comment]
hey sorry it came late but i finally figured out a way to do it; i just came across this issue while researching on a similar problem.


the solution:

const intersects = this.raycaster.intersectObject(player.children[3], true);

where player is the group object and [3] is the id of the main object i'm trying to ray cast,which in my case is a 3d model
please note this only works if the object is a 3d obejct with geometry.

i'm available for further questions
